styles/style.css

```diff
@@ -0,0 +1,148 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
+
+body {
+    text-align: center;
+    margin-right: 5%;
+    margin-left: 5%;
+    font-family: sans-serif;
+}
+
+body div > section {
+    border-bottom: solid lightgray;
+    margin-top: 5%;
+    padding-bottom: 5%;
+}
+  
+ul {
+    list-style-type: none;
+    margin-left: 0px;
+    padding-left: 0px;
+}
+
+nav ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+}
+
+nav ul li {
+    padding: 5%;
+    font-size: 80%;
+    font-family: 'Open Sans',serif;
+}
+
+nav ul :first-child {
+    margin-right: auto;
+}
+  
+#page_one {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 5%;
+    line-height: 500%;
+}
+
+#page_one sub {
+    font-size: 150%;
+    margin-bottom: 5%
+}
+
+#page_one a {
+    text-decoration: none;
+    color: black;
+    border-radius: 5%;
+    border: solid lightgray;
+    width: 200px;
+    align-self: center;
+}
+
+#why h2 {
+    font-size: 2rem;
+}
+
+#answerstowhy {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+}
+
+#answerstowhy li {
+    display:flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    width: 30%;
+    padding: 3%;
+}
+
+#answerstowhy img {
+    height: 100px;
+    width: 100px;
+    float: left;
+    padding-right: 1% 1% 0 0;
+}
+
+.reason {
+    text-align: left;
+    padding: 1%;
+}
+
+.quote {
+    font-style: italic;
+    line-height: 200%;
+}
+
+.quote blockquote {
+    text-align: center;
+}
+
+.quote p {
+    text-align: right;
+}
+
+.quote p:before {
+    content: "—";
+}
+
+#meettheteam ul {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+
+#meettheteam ul li{
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    text-align: center;
+    padding: 8%;
+}
+
+#meettheteam img {
+    border-radius: 50%;
+    flex-wrap: wrap;
+}
+
+#sponsors ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    flex-wrap: wrap;
+    margin-left: 2%;
+    margin-right: 2%;
+}
+
+#sponsor ul li {
+    padding: 1%;
+}
+
+#companyinfo {
+    display: grid;
+    grid-template-columns: 1fr 3fr;
+    grid-template-rows: 5% 5% 5% 5%;
```

Comment:
This is a super smart approach to making the company info lay out on a grid. Using the `1fr 3fr` helped create all that white space on the side.